apis/v1beta1/types.go

```diff
@@ -258,6 +258,11 @@ type PCIDeviceSpec struct {
 	// virtual machine is cloned.
 	// +kubebuilder:validation:Required
 	VendorID *int32 `json:"vendorId,omitempty"`
+	// Label is the hardware label of a virtual machine's PCI, in string.
```

Comment:
```suggestion
	// CustomLabel is the hardware label of a virtual machine's PCI device.
	// It allows to restrict the placement to specific hardware instances.
```

Does this map to this docs: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-5B3CAB26-5D06-4A99-92A0-3A04C69CE64B.html ?

Reply:
Actually, no, I came up with that myself 😄 
Fixed.